dev/ci/docker_linux/Dockerfile

```diff
@@ -124,11 +124,18 @@ RUN $GOBINARY get -u github.com/technosophos/dashing
 RUN locale-gen en_US "en_US.UTF-8" && dpkg-reconfigure locales
 ENV LANG en_US.UTF-8
 
+# Install fastlane via pinned Gemfile & Gemfile.lock
 # Skip all the documentation (-N) since it's just on CI.
 RUN gem install bundler -N
 
 # Gemfile last edited 1/6/2019 (update to trigger a rebuild of Docker image)
-COPY Gemfile /Gemfile
-COPY Gemfile.lock /Gemfile.lock
+COPY ci/docker_linux/Gemfile /Gemfile
+COPY ci/docker_linux/Gemfile.lock /Gemfile.lock
 
 RUN bundle install --system
+
+# Install goldctl, for Golden testing
+# Last updated 1/8/19 (update to rebuild Dockerfile with latest goldctl)
```

Comment:
While you're at it, can you update this date, so we can cache bust and get the newest goldctl?

Reply:
Done.